tests/distributions/multivariate/test_laplace.py

```diff
@@ -0,0 +1,207 @@
+    rv = dist.dist(mu=mu[0], chol=chol, lower=False)
+    mu_param, cov_param = rv.owner.op.dist_params(rv.owner)
+    # It's a bit silly but we transpose twice when lower=False
+    cov_expected = chol.T @ chol.T.T
+    mu_expected, _ = pt.broadcast_arrays(mu[0], cov_expected[..., -1])
+    assert equal_computations([mu_param, cov_param], [mu_expected, cov_expected])
+
+    # Mu and tau
+    rv = dist.dist(mu=mu, tau=tau)
+    mu_param, cov_param = rv.owner.op.dist_params(rv.owner)
+    assert equal_computations([mu_param, cov_param], [mu, pt.linalg.inv(tau)])
+
+
+@pytest.mark.parametrize(
+    "mu, cov, size, expected",
+    [
+        (np.arange(6).reshape(3, 2), np.eye(2), None, np.arange(1, 7).reshape(3, 2)),
+        (
+            np.arange(2),
+            np.broadcast_to(np.eye(2), (3, 2, 2)),
+            None,
+            np.broadcast_to(np.arange(1, 3), (3, 2)),
+        ),
+        (0, np.eye(3), (4,), np.ones((4, 3))),
+    ],
+)
+@pytest.mark.parametrize("dist", [MvLaplace, MvAsymmetricLaplace])
+def test_support_point(dist, mu, cov, size, expected):
+    with pm.Model() as model:
+        dist("x", mu=mu, cov=cov, size=size)
+    assert_support_point_is_expected(model, expected)
+
+
+@pytest.mark.parametrize("dist", [MvLaplace, MvAsymmetricLaplace])
+def test_mean(dist):
+    mu = [np.pi, np.e]
+
+    # Explicit size
+    rv = dist.dist(mu=mu, chol=np.eye(2), size=(3,))
+    mean = pm.distributions.moments.mean(rv)
+    np.testing.assert_allclose(mean.eval(mode="FAST_COMPILE"), np.broadcast_to(mu, (3, 2)))
+
+    # Implicit size from cov
+    rv = dist.dist(mu=mu, cov=np.broadcast_to(np.eye(2), (4, 2, 2)))
+    mean = pm.distributions.moments.mean(rv)
+    np.testing.assert_allclose(mean.eval(mode="FAST_COMPILE"), np.broadcast_to(mu, (4, 2)))
+
+
+@pytest.mark.parametrize("dist", [MvLaplace, MvAsymmetricLaplace])
+def test_random(dist):
+    mu = [-1, np.pi, 1]
+    cov = [[1, 0.5, 0.25], [0.5, 2, 0.5], [0.25, 0.5, 3]]
+    rv = dist.dist(mu=mu, cov=cov, size=10_000)
+
+    samples = pm.draw(rv, random_seed=13)
+    assert samples.shape == (10_000, 3)
+    np.testing.assert_allclose(np.mean(samples, axis=0), mu, rtol=0.05)
+
+    expected_cov = cov if dist is MvLaplace else cov + np.outer(mu, mu)
+    np.testing.assert_allclose(np.cov(samples, rowvar=False), expected_cov, rtol=0.1)
+
+
+def test_symmetric_matches_univariate_logp():
+    # Test MvLaplace matches Univariate Laplace when there's a single entry
+    mean = 1
+    scale = 2
+    # Variance of Laplace is 2 * scale ** 2
+    rv = MvLaplace.dist(mu=[mean], cov=[[2 * scale**2]])
+    ref_rv = pm.Laplace.dist(mu=mean, b=scale)
+
+    test_val = np.random.normal(size=(3, 1))
+    rv_logp = pm.logp(rv, test_val).eval()
+    ref_logp = pm.logp(ref_rv, test_val).squeeze(-1).eval()
+    np.testing.assert_allclose(rv_logp, ref_logp)
+
+
+@pytest.mark.xfail(reason="Not sure about equivalence. Test fails")
+def test_asymmetric_matches_univariate_logp():
```

Comment:
I was expecting these to be equivalent. The means and variances match, but not the logp. Either I did something wrong or the equivalence is not real / like this.